content/blog/2022/turing-open-service-questions/index.md

```diff
@@ -0,0 +1,43 @@
+---
+title: "Three open questions for building open services"
+subtitle: ""
+summary: ""
+authors: ["Chris Holdgraf"]
+tags: []
+categories: [updates]
+date: 2022-07-15
+featured: false
+draft: false
+---
+
+# Three open questions for building open services
+
+Over the past few weeks I've had an opportunity to brainstorm with members from [The Turing Way](https://the-turing-way.netlify.app/welcome) and [The Turing Institute](https://www.turing.ac.uk/) about how 2i2c can refine is collaborative service model to be more impactful.
+We recently had a day of brainstorming with [Malvika Sharan](https://github.com/malvikasharan), [Alden Conner](https://github.com/aldenc), [Hari Sood](https://github.com/harisood), and [Arielle Bennett](https://github.com/Arielle-Bennett).
+One product that came out of this meeting was three important questions that we need to answer.
+The reflect several major questions around creating community-centered services that strike the right division of labor and responsibility for each partner in the service.
+
+Some of our discussions around this can also be found in [this GitHub Issue](https://github.com/2i2c-org/team-compass/issues/446).
+
+## Where is the line between 2i2c and communities?
+
+- We want to build spaces for communities to interact with us, but do not want communities to lose their own identity
+- Example: communication forums
+- How do we avoid colonizing them
+- What if they don't have a pre defined community structure? Or are small? 
+- When is it time for them to move on and create their own space?
+
+## What can we ask of communities?
+
+- We treat each hub as a collaboration
+- We can provide infrastructure and guidance around it, but communities know what is best for themselves
+- What can we ask of them in collaborating with us in the service? What subsets of people could take on more work than others? Post docs? Probably not PIs
```

Comment:
You are assuming there are post-docs and PIs... maybe that is common in our communities but I can see other structures as well in other sorts of communities.